Mathlib/Analysis/SpecificLimits/Normed.lean

```diff
@@ -538,6 +538,32 @@ theorem mul_neg_geom_series (x : R) (h : ‖x‖ < 1) : ((1 - x) * ∑' i : ℕ,
   rw [← mul_neg_geom_sum, Finset.mul_sum]
 #align mul_neg_geom_series mul_neg_geom_series
 
+theorem geom_series_succ (x : R) (h : ‖x‖ < 1) : ∑' i : ℕ, x ^ (i + 1) = ∑' i : ℕ, x ^ i - 1 := by
+  apply symm
+  rw [sub_eq_iff_eq_add, tsum_eq_zero_add (NormedRing.summable_geometric_of_norm_lt_one x h),
+    pow_zero, add_comm]
+
+theorem geom_series_mul_shift (x : R) (h : ‖x‖ < 1) :
+    x * ∑' i : ℕ, x ^ i = ∑' i : ℕ, x ^ (i + 1) := by
```

Comment:
`h` is not really necessary here (if it doesn't hold, then both sides are zero by definition).

Reply:
(But it appears that the relevant statement wants a division ring, so it's probably not worth removing the condition.)